applications/FluidDynamicsApplication/tests/FluidRVETest/fluid_rve_test_materials_2D.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think that we can have a single json file and edit the constitutive law name according to the problem DOMAIN_SIZE as you do with the other settings.

Reply:
Done as well.

Reply:
In the end it was left as it was given that, when opening the materials json as a Kratos parameters object what's being edited (constitutive_law, name field) is the object, not the json.